### DISCUSSION TITLE
🔗 Loop builder, add support for category loop (Duplicate of Issue #20018)

### DISCUSSION THREAD

**Discussion by Elsteck** [elementor/elementor#21178] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

It is great to have a loop builder now, but it only loops through posts and products and additional plugins are still needed for a very basic loop function: creating an archive page or grid with a loop for categories. This is often used as an alternative visual navigation.

### Describe the solution you'd like

Add a query for categories to the loop builder

### Describe alternatives you've considered

Other addons that offer this feature are quite heavy, JetEngine, thePlusaddons, Powerpack, Dynamic Content for Elementor

**Comment by black-eye** [comment]
+1
This is absolutely essential!

**Comment by nicholaszein** [comment]
Hello there!

Thank you for your submission! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here #20018

👉 Feel free to continue the conversation there!

Kind regards,

Nick
